modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/PolicyDetails/policy-summary-details/index.tsx

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import Summary from '../../../../../common/components/summary';
+
+type SummaryType = {
+	summaryData: {
+		applicationDataJSON: string;
+		boundDate: string;
+		commission: number;
+		email: string;
+		endDate: string;
+		phone: string;
+		policyDataJSON: JSON;
+		termPremium: number;
+	};
+};
+const PolicySummary = ({summaryData}: SummaryType) => {
+	const applicationDataJSON = summaryData.applicationDataJSON
+		? JSON.parse(summaryData.applicationDataJSON)
+		: {};
+
+	const {driverInfo} = applicationDataJSON;
+
+	const policyEndDate = Date.parse(summaryData?.endDate);
+
+	const differenceOfDays = Math.abs(+policyEndDate - +new Date());
+
+	const renewalDue = Math.floor(differenceOfDays / (1000 * 60 * 60 * 24)) + 1;
+
+	const totalClaimAmount = 1963.58;
+
+	const coverageLimit = '$2,500.00/$100,000.00';
+
+	function dateFormatter(date: string) {
+		const formattedDate = new Date(date).toLocaleDateString('en-us', {
+			day: '2-digit',
+			month: '2-digit',
+			timeZone: 'UTC',
+			year: 'numeric',
+		});
+
+		return formattedDate;
+	}
+
+	function valueFormatter(value: number) {
+		const dollarValue = Intl.NumberFormat('en-US', {
+			currency: 'USD',
+			style: 'currency',
+		});
+
+		return dollarValue.format(value);
+	}
+
+	const summaryPolicyData = [
```

Comment:
Can you put it inside an useMemo?

I can help to avoid loops and unnecessary recreation of the data